[PATCH] ltrace: fix build when USE_DEMANGLE isn't enabled

summary.c unconditionally calls my_demangle(), which is only available
when USE_DEMANGLE is enabled (which requires libiberty).

Signed-off-by: Peter Korsgaard <jacmet@sunsite.dk>
---
 summary.c |    5 +++++
 1 file changed, 5 insertions(+)

Index: ltrace-0.5/summary.c
===================================================================
--- ltrace-0.5.orig/summary.c
+++ ltrace-0.5/summary.c
@@ -83,7 +83,12 @@ void show_summary(void)
 		       (int)entries[i].tv.tv_sec, (int)entries[i].tv.tv_usec,
 		       (unsigned long int)(c / entries[i].count),
 		       entries[i].count,
+#ifdef USE_DEMANGLE
 		       opt_C ? my_demangle(entries[i].name) : entries[i].name);
+#else
+		       entries[i].name);
+#endif
+
 	}
 	printf
 	    ("------ ----------- ----------- --------- --------------------\n");
